client/src/components/Game/Endgame.js

```diff
@@ -24,18 +26,34 @@ class Endgame extends Component {
     this.fetchGame(id);
   }
 
+   async idToUsername(players) {
+    // For the MVP, we won't listen for username changes
+    // TODO add listener in Project Alpha
```

Comment:
Is an Issue created for this? If so can you reference it here?

Reply:
could you link me to where this happens in your repo so i have an idea of the syntax